packages/modal/src/Modal.styled.js

```diff
@@ -0,0 +1,56 @@
+import styled from "styled-components";
+import { animated } from "react-spring";
+import { colors } from "@crave/farmblocks-theme";
+import Card from "@crave/farmblocks-card";
+
+export const Wrapper = styled(animated.div)`
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  padding: 40px;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
```

Comment:
We can configure this setting by passing a `className` to the Modal and change it accordingly. 
Taking the `With custom parentNode` as an example, we would have this:

```diff
.add("With custom parentNode", () => {
    const Parent = styled.div`
      border: dashed 2px gray;
      width: 600px;
      height: 400px;
      padding: 10px;
      margin: 40px;
      position: relative;

+     .center {
+       justify-content: center;
+     }
    `;
    const Example = () => {
      const [isOpen, setIsOpen] = useState(false);
      const parentRef = useRef(null);

      return (
        <div>
          <button onClick={() => setIsOpen(true)}>Open Modal</button>
          <Modal
            parentNode={parentRef.current}
            isOpen={isOpen}
            onRequestClose={() => setIsOpen(false)}
+           className="center"
          >
            Hello.
          </Modal>

          <Parent ref={parentRef}>The Modal will render here</Parent>
        </div>
      );
    };
    return <Example />;
  });
```

Maybe we could allow this customisation via props, what do you think?